### ISSUE TITLE
Screen reader sections for HTML tags

### ISSUE THREAD

**Issue by estebanz01** [mdn/browser-compat-data#6553]
Hello! please, let me know if this is not the place for a _feature request_ so I can close it properly.

I've been working the last couple of years on making a platform WCAG compliant with version 2.0 and 2.1 and often times we have to rely on QA to know if an HTML/CSS feature is screen reader friendly and it outputs the desired behaviour. This is time consuming and sometimes it leads to live bugs that needs to get, again, addressed and checked against the final QA step to see if it works or not.

So, I was wondering if it's possible to have a section for screen readers that display something like this:

| feature | Voice Over | NVDA | JAWS |
|  -----------: |  -----------: |  -----------: |  -----------: |
| details status | Does not read expanded / collapsed | Reads expanded / collapsed | Reads expanded / collapsed |
| summary | Reads summary as group title | Reads summary as group title | Reads summary as group title |

Thanks!

**Comment by estebanz01** [comment]
Amazing resource @Malvoz Thanks!

**Comment by Elchi3** [comment]
Closing this as we are currently not planning to also record compat data for screen readers in BCD. https://a11ysupport.io/ is a great project for that and showing it on MDN should be discussed in https://github.com/mdn/yari (as tried in https://github.com/mdn/yari/issues/4454)
